jetpack/jetpack-start-endpoints/plan-provisioning.md

```diff
@@ -102,7 +17,7 @@ Plans can be provisioned by making a request using your partner token from the s
 
 - __local_user__:      The username, ID or email on the local website (not the WordPress.com username) that should own the plan. The corresponding user _must_ already exist.
 - __siteurl__:         The URL where the WordPress core files reside.
-- __plan__:            A slug representing which plan to provision. One of `free`, `personal`, `premium`, or `professional`.
+- __plan__:            A slug representing which plan or item to provision. One of `free`, `personal`, `premium`, or `professional` for plans. For backups either `jetpack-backup-realtime` or `jetpack-baackup-daily`
```

Comment:
There is a typo in `jetpack-baackup-daily`.

Also, I'd probably add a comma after "For backups".